src/main/resources/crafter/engine/server-config.properties

```diff
@@ -289,6 +289,18 @@ crafter.engine.site.cache.warmUp.content.folders=/scripts,/templates
 ####################
 # The site name parameter or cookie name, used when resolving the current site
 crafter.engine.request.param.siteName=crafterSite
+# Indicates if the cookie should be updated if missing or outdated
```

Comment:
Change to `crafter.engine.response` since all these properties are for setting the cookie on the response.